src/main/java/nl/igorski/example/MWEngineActivity.java

```diff
@@ -39,7 +39,7 @@
     private Vector<SynthEvent>  _synth1Events;
     private Vector<SynthEvent>  _synth2Events;
     private Vector<SampleEvent> _drumEvents;
-    private SynthEvent          _liveEvent;
+    private SampleEvent         _liveEvent;
 
```

Comment:
Temporary alteration of the example activity just for testing purposes of this feature.